libs/designer/src/lib/ui/panel/panelRoot.tsx

```diff
@@ -28,15 +28,44 @@ const layerProps = {
   eventBubblingEnabled: true,
 };
 
+const useStyles = makeStyles({
```

Comment:
Has this been tested with panel on the left also?

Reply:
Hi @ek68794998, yep. As for now this feature will only support right side panels.

Reply:
Cool, thanks for keeping that in mind!